setup.py

```diff
@@ -4,7 +4,7 @@
 from setuptools import setup, find_packages
 setup(
     name = "a10-neutronclient",
-    version = "0.1.5a",
+    version = "0.1.6a",
```

Comment:
Don't bump versions as part of functionality changes. We'll just end up double-bumping.